### ISSUE TITLE
List Every Integration in Artifact Hub

### ISSUE THREAD

**Issue by agardnerIT** [keptn-contrib/artifacthub#94]
Currently Keptn has 4 potential places that tooling integration can be found:

1. Keptn core 
2. [keptn-contrib](https://github.com/keptn-contrib)
2. [keptn-sandbox](https://github.com/keptn-sandbox)
3. artifactHub (this repo)

Users of Keptn won't understand this and will just look at the Integrations page of Keptn.sh and see very few tooling integrations.

This issue suggests that **all** proven and actively maintained tooling integrations should be listed on artifactHub as the "one stop shop".

If integrations are still in incubating status, perhaps a warning is added to artifacthub?

| Integration                    | ArtifactHub   | Keptn Contrib | Sandbox | Status        |
| -------------------------- | ------------- | --------------- | --------- |--------------|
| Argo                              | 🟢                |     🟢              |               |     ✅  OK    |
| Artillery                          | 🟢                |                       |       🟢   |     ✅  OK    |
| Dynatrace                      | 🟢                |     🟢             |                |     ✅  OK    |
| GitHub   Actions            | 🟢                | 🟢                 |                |   ✅  OK    |
| Helm                              |                      |                       |                |   ✅  OK  (Keptn core)  |
| IFTTT                              | 🟢               |                        |                |     ✅  OK    |
| Jenkins                           | 🟢                |                      |  🟢          |    ✅  OK    |
| JIRA                                | 🟢                |                     | 🟢            |   ✅  OK    |
| Jmeter                            |                      |                      |                 |   ✅  OK   (Keptn core) |
| Job Executor Service      | 🟢                |  🟢               |                 |   ✅  OK   |
| Litmus                            |  🟢                 |                      |   🟢         |     ✅  OK   |
| Locust                            |    🟢                |                    |    🟢        |     ✅  OK   |
| Monaco                         |     🟢                |                   | 🟢           |    ✅  OK   |
| NeoLoad                       |  🟢                   | 🟢           |                   |    ✅  OK   |
| Prometheus                  |  🟢                   | 🟢            |                  |   ✅  OK   |
| ServiceNow                  | 🟢                    |  🟢            |                 |   ✅  OK   |
| Slack                             | 🟢                   |                   | 🟢            |   ✅  OK   |
| Unleash                        | 🟢                   | 🟢              |                 |   ✅  OK   |
| Zapier                           |  🟢                  |                     |                |   ✅  OK   |
| Notification Service      |                         | 🟢              |                 | ⚠️            |
| Datadog                       |                          |                   | 🟢           | ⚠️             |
| GitOps Operator          |                          |                   | 🟢           | ⚠️             |
| AWS FIS                       |                          |                   | 🟢           | ⚠️              |
| AWS CodeDeploy        |                         |                   | 🟢            | ⚠️              |
| Akamas                       |                         |                     | 🟢          | ⚠️                 |
| Django Migration       |                         |                    | 🟢           | ⚠️              |
| Crossplane                  |                         |                  | 🟢           | ⚠️              |
| OneChart (gimlet)       |                        |                  | 🟢           | ⚠️              |
| Zendesk                      |                        |                  | 🟢           | ⚠️              |
| Grafana                       |                        |                  | 🟢           | ⚠️              |
| xmatters                     |                         |                  | 🟢           | ⚠️              |
| Azure DevOps            |                         |                  | 🟢           | ⚠️              |
| mysupermon              |                         |                  | 🟢           | ⚠️              |
| backstage                  |                          |                  | 🟢           | ⚠️              |
| Falco                         |                           |                  | 🟢           | ⚠️              |
| Ansible Tower            |                          |                  | 🟢           | ⚠️              |
| GitLab                        |                          |                  | 🟢           | ⚠️              |
| Selenium                   |                          |                  | 🟢           | ⚠️              |
| Alexa                         |                          |                  | 🟢           | ⚠️              |
| Wavefront                 |                          |                  | 🟢           | ⚠️              |
| Blazemeter                |                          |                  | 🟢           | ⚠️              |
| Trello                         |                          |                  | 🟢           | ⚠️              |
| OsTicket                    |                          |                  | 🟢           | ⚠️              |
| Monday.com             |                          |                  | 🟢           | ⚠️ #93       |

</body>

</html>

**Comment by christian-kreuzberger-dtx** [comment]
Hi, and thanks for this overview. The idea of the artifacthub repo is to list all integrations, as you mentioned.

The idea is that every owner can enter their service, and especially if you have a dedicated GitHub repo for it, we have some automation (e.g., a new release of prometheus-service will automatically create a Pull Request in Artifacthub)

There are some exceptions that I would like to mention:

* helm-service and jmeter-service are built-in with Keptn core - we haven't listed them for now, as they are part of the Keptn core installation and part of Keptn core use-cases (same as e.g., lighthouse-service)
* Notification Service (among others) is not maintained. We are not listing integrations that haven't received updates in a long time and that are probably incompatible.

There are some more integrations that I'm not too happy with being listed, e.g., litmus-service and locust-service. 
Locust-service can easily be replaced with a proper job-executor-service example (we already have one here: https://github.com/keptn-contrib/job-executor-service/blob/main/FEATURES.md#kubernetes-job
Litmus-service is not maintained at all, ... 

Last but not least, datadog-service: That's a rather new integration. If it works, we are happy to accept it to the artifacthub repo.

**Long story short:** We should provide our end-users the best possible experience by offering **working** integrations on ArtifactHub. We are not locking anyone out, but we are not pro-actively adding **old** and most likely **non-working** integrations.
